### ISSUE TITLE
Problem accessing '/1.1/user.json

### ISSUE THREAD

**Issue by fr1tzz** [DTVD/rainbowstream#174]
```
We have connection problem with twitter stream API right now :(
<html>\n<head>\n<meta http-equiv="Content-Type" content="text/html; charset=utf-8"/>\n<title>Error 401 Unauthorized</title>
</head>
<body>
<h2>HTTP ERROR: 401</h2>
<p>Problem accessing '/1.1/user.json'. Reason:
<pre>    Unauthorized</pre>
</body>
</html>
```

Ive tried both the quick way and the venv way. Do I need to redo my auth? How would I do that?

I am on Debain 8.4 Jessie and use python 3.4

**Comment by DTVD** [comment]
Connection to Twitter API may unstable. You ca redo the auth by deleting your `~/.rainbow_oauth` file and start RainbowStream again

**Comment by fr1tzz** [comment]
I have tried this and the same error occurs. Is there some trouble shooting steps I can do?

**Comment by DTVD** [comment]
Can you go to your Twitter account setting : https://twitter.com/settings/applications and check how RainbowStream displayed there ? This is how it should be displayed
![](https://i.gyazo.com/a10acf5b15fc199fe3e29debb53ebb09.png)

**Comment by fr1tzz** [comment]
It is the same as yours. Could it be something else?


![selection_001](https://cloud.githubusercontent.com/assets/19253855/15200872/0de12d30-17b9-11e6-85fd-0e751f1eb121.png)

**Comment by DTVD** [comment]
It seems that nothing could be wrong... 
Can you Revoke access + delete `~/.rainbow_oauth` file and start again ?

**Comment by fr1tzz** [comment]
I have tried this and the same error is still occurring. Would you like me to send you any logs or files to help you fix this issue/bug? here is the output.
![selection_003](https://cloud.githubusercontent.com/assets/19253855/15228950/8fbe846c-185c-11e6-9f0c-3888bc0455a5.png)

**Comment by DTVD** [comment]
This is error from an underlying library which RaibowStream is using, so I can not know any more detail. Could you try the following:
* Use another 3rd party Twitter app (not thoses come from Twitter like Twitter web, Twitter official IOS/Android app or TweetDeck) with _same account_.
* Use _another differect account_ with RainbowStream.and above app.

And tell which is work and which is not ? Thanks!

**Comment by fr1tzz** [comment]
UPDATE/RESOLVED

After giving up and trying all sorts of things to fix this issue I've finally resolved it, although I'm not sure what exactly was causing this. Maybe some one else can tell me or anyone else who needs to know for similar issues what was wrong I think it had something to do with outdated packages.

Anyway, To fix this on DEBIAN I did the following, 
1. updated my repository to the testing branch
2. apt-get update
3. apt-get upgrade
4. apt-get autoremove
5. apt-get dist-upgrade
After that I rebooted and had no further issues. I hope this helps somebody.
